fdbserver/Status.actor.cpp

```diff
@@ -2346,11 +2372,17 @@ ACTOR Future<StatusReply> clusterGetStatus(
 				statusObj["fault_tolerance"] = faultToleranceStatusFetcher(configuration.get(), coordinators, workers, extraTlogEligibleZones, minReplicasRemaining, loadResult.present() && loadResult.get().healthyZone.present());
 			}
 
-			JsonBuilderObject configObj = configurationFetcher(configuration, coordinators, &status_incomplete_reasons);
+			state JsonBuilderObject configObj =
+			    configurationFetcher(configuration, coordinators, &status_incomplete_reasons);
 
 			// configArr could be empty
-			if (!configObj.empty())
+			if (!configObj.empty()) {
+				Optional<Value> primaryDCO = wait(getActivePrimaryDC(cx));
```

Comment:
It just occurred to me that we should attempt to run this concurrently with some of our other status fetches (maybe in the `futures2` vector).